### DISCUSSION TITLE
Joining a window with another in accordion layout

### DISCUSSION THREAD

**Discussion by gservat** [nikitabobko/AeroSpace#704] [questions-and-answers]
Hi,

Thanks @nikitabobko for enabling discussions. I hope they have reduced some of the noise in issues that should be questions (and vice versa).

I am running accordion layout in one of my workspaces where I have several VS Code windows. I sometimes want to have two windows side-by-side in this accordion layout. I can't switch to tiles as there's many windows so each window will be barely visible.

My workaround currently is to move the windows that I want to put side-by-side to another workspace that has tiles layout.

I might be the only one that wants to do this but thought I'd ask 😄 

Thanks!!

**Comment by alextremblay** [comment]
you're not the only one.

I use accordion layout almost exclusively, and i sometimes have many windows in a given workspace, to the point where any other layout just doesn't work.

I'd love to have the ability to "pop out" a single window with a keybind, and then "pop in" that same window when I'm done

more specifically, starting with a layout like this:
```
h_accordion
├── window 1
├── window 2 (focused)
├── window 3
└── window 4
```

and hitting a keybind, i'd like to end up in a layout like this:

```
h_tiles
├── h_accordion
|   ├── window 1
|   ├── window 3
|   └── window 4
└── window 2 (focused)
```

and then hitting the same keybind, I'd like to end up back at the original layout.


Anybody know how to make that happen?
